server.js

```diff
@@ -49,6 +49,76 @@ app.use(json({
 const partitioningRoutes = require('./privacy-protections/storage-partitioning/server/routes');
 app.use('/partitioning', partitioningRoutes);
 
+function joinPath () {
```

Comment:
Is it possible to move these into the adClickFlow routes.js?

Reply:
Not really, these rewrite the path to the relevant _/adClickFlow_ path. The /adClickFlow/server/routes.js is applied only to that path with _app.use('/adClickFlow', adClickFlowRoutes);_


The only other way here is export a function from the server path; I'd like to punt this to a fast follow commit that has a separate approval as we've already been testing this well.